source/extensions/common/aws/utility.cc

```diff
@@ -11,6 +12,14 @@ namespace Extensions {
 namespace Common {
 namespace Aws {
 
+static const char PATH_SPLITTER = '/';
```

Comment:
I would just make all of these all into `static constexpr absl::string_view` in some places you are having to allocate strings for the chars which is unnecessary.